flipperdevices/flipperzero-firmware
# [FL-3346] fbt: added Flipper selection when multiple are connected over USB

# What's new

- [FL-3346] fbt: added Flipper selection when multiple are connected over USB
- Also fixes #2722

# Verification 

- Connect more than one fiipper over USB
- Check that you can select which one to use for `fbt cli`, `fbt flash_usb`, etc by adding `FLIP_PORT=/dev/cu....`

# Checklist (For Reviewer)



[FL-3346]: https://flipperzero.atlassian.net/browse/FL-3346?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
